conf/supervisord-dev.conf

```diff
@@ -0,0 +1,50 @@
+[supervisord]
+nodaemon = true
+
+[program:web]
+command = tox -qqe dev --run-command 'pserve --reload conf/development-app.ini'
+stdout_events_enabled=true
+stderr_events_enabled=true
+stopsignal = KILL
+stopasgroup = true
+killasgroup = true
```

Comment:
Supervisor fails to kill `pserve` properly without these. Probably not every program strictly needs these but I think it's best to play it safe and always use them

Reply:
Done